[READY] Removing NeoBundle mentions from README

Valloric opened the pull request:
NeoBundle has been abandoned.

Also deleting the super-obsolete Bitdeli badge.

Fixes #2208.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/valloric/youcompleteme/2210)
<!-- Reviewable:end -->

puremourning commented:
Reviewed 3 of 3 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[third_party/ycmd, line 1 \[r1\]](https://reviewable.io:443/reviews/valloric/youcompleteme/2210#-KK_YymvSVopKi-0xTzc:-KK_YymvSVopKi-0xTzd:966464229) ([raw file](https://github.com/valloric/youcompleteme/blob/f7a1e8e341415d08eb57f469bd6154b8fb73a6bf/third_party/ycmd#L1)):*
> ```
> Subproject commit ad41ee4bff4997ef4c411988d30c344ee3fea5b7
> ```

Is the ycmd update here intentional? should it strictly be part of this CR?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/valloric/youcompleteme/2210)*
<!-- Sent from Reviewable.io -->

puremourning commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> assuming the ycmd update is intentional P:)

---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/valloric/youcompleteme/2210#-:-KK_Z4jYrK1td9wu98D2:-321551537)*
<!-- Sent from Reviewable.io -->

Valloric commented:
It's not intentional; in fact, good that you caught that!

vheon commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> Don't know why Reviewable is still showing they ycmd submodule even when there is not change :confused: 

---

Reviewed 2 of 3 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/valloric/youcompleteme/2210#-:-KK_qxL4igQNbKdkCraO:-1940935628)*
<!-- Sent from Reviewable.io -->

mispencer commented:
*[doc/youcompleteme.txt, line 3139 \[r1\]](https://reviewable.io:443/reviews/valloric/youcompleteme/2210#-KKa2DyYsnjnH4ZDMhqo:-KKa2DyYsnjnH4ZDMhqp:-998241974) ([raw file](https://github.com/valloric/youcompleteme/blob/f7a1e8e341415d08eb57f469bd6154b8fb73a6bf/doc/youcompleteme.txt#L3139)):*
> ```
> YouCompleteMe contributors
> 
>   Image: Bitdeli Badge [66]
> ```

Is this image still supposed to be here?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/valloric/youcompleteme/2210)*
<!-- Sent from Reviewable.io -->

Valloric commented:
Review status: 1 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[doc/youcompleteme.txt, line 3139 \[r1\]](https://reviewable.io:443/reviews/valloric/youcompleteme/2210#-KKa2DyYsnjnH4ZDMhqo:-KKaF8NtxWrMfXTjybtf:-527824112) ([raw file](https://github.com/valloric/youcompleteme/blob/f7a1e8e341415d08eb57f469bd6154b8fb73a6bf/doc/youcompleteme.txt#L3139)):*
<details><summary><i>Previously, mispencer wrote…</i></summary>
> Is this image still supposed to be here?
</details>
Fixed now, good catch!

---

*[third_party/ycmd, line 1 \[r1\]](https://reviewable.io:443/reviews/valloric/youcompleteme/2210#-KK_YymvSVopKi-0xTzc:-KKaF6T9y0RlYXLwIjmI:-590542770) ([raw file](https://github.com/valloric/youcompleteme/blob/f7a1e8e341415d08eb57f469bd6154b8fb73a6bf/third_party/ycmd#L1)):*
<details><summary><i>Previously, puremourning (Ben Jackson) wrote…</i></summary>
> Is the ycmd update here intentional? should it strictly be part of this CR?
</details>
It was not, removed.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/valloric/youcompleteme/2210)*
<!-- Sent from Reviewable.io -->

micbou commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

@homu r+

---

Reviewed 1 of 3 files at r1, 1 of 1 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/valloric/youcompleteme/2210#-:-KKbzmUBO_vkqsTe5ab6:187772251)*
<!-- Sent from Reviewable.io -->

homu commented:
:pushpin: Commit 9c287cd has been approved by `micbou`

<!-- @homu r=micbou 9c287cde91d4de91e0fa682d2b57fc8930ca9b02 -->

homu commented:
:zap: Test exempted - [status](https://travis-ci.org/Valloric/YouCompleteMe/builds/138654768)
